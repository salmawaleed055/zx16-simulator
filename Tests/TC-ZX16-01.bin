java -jar rars.jar tc-zx16-10.s --dump .text Binary tests/TC-ZX16-10.bin
