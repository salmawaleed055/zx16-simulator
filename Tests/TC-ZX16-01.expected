x1 = 0x0005
x2 = 0x0007
x3 = 0x000C
